BUILD.md

```diff
@@ -17,15 +17,6 @@ The basic build uses docker to pull down its toolchain and compile.
 
```

Comment:
@paulwizviz as the very first step i think you need to include export ISOLATION_ID=....